App/Sagas/NodeLifecycle.ts

```diff
@@ -236,31 +222,22 @@ function * backgroundTaskRace () {
 
 function * stopNodeAfterDelay (ms: number) {
   try {
-    if (yield select(PreferencesSelectors.verboseUi)) {
-      yield call(displayNotification, 'Running the node for 20 sec. in the background')
-    }
+
+    yield put(MockBridgeActions.stopNodeAfterDelayStarting())
```

Comment:
Good example of an event well need to create from the SDK that is just mocked here by mockbridge